static/app/components/tabs/tab.tsx

```diff
@@ -138,6 +160,44 @@ export const Tab = forwardRef(
   }
 );
 
+const NewTabWrap = styled('li', {shouldForwardProp: tabsShouldForwardProp})<{
+  isTempTab: boolean;
+  overflowing: boolean;
+  selected: boolean;
+}>`
+  ${p =>
+    p.selected
```

Comment:
Instead of using the prop, it is better to use attributes when applicable since it does not require styles to be generated dynamically:

```css
&[aria-selected="true"] {
  /* selected styles */
}
```